site/app/models/gradeable/Gradeable.php

```diff
@@ -159,34 +69,34 @@ class Gradeable extends AbstractModel {
     private $db_components = [];
 
     /** @prop
-     * @var bool If any submitters have active grade inquiries */
+     * @var int If any submitters have active grade inquiries */
     protected $active_grade_inquiries_count = 0;
 
     /* (private) Lazy-loaded Properties */
 
     /** @prop
-     * @var bool If any manual grades have been entered for this gradeable */
+     * @var ?bool If any manual grades have been entered for this gradeable */
     protected $any_manual_grades = null;
```

Comment:
This change and similar changes below doesn't make sense.  This variable is being initialized to `null`, yet you're removing the nullable type annotation.

Reply:
Looking at this again, I'm not sure what I was thinking when I left this comment...  This change is correct.